RadarSDK/RadarLocationManager.m

```diff
@@ -871,6 +871,9 @@ - (void)sendLocation:(CLLocation *)location stopped:(BOOL)stopped source:(RadarL
                                      completionHandler:^(RadarStatus status, NSDictionary *_Nullable res, NSArray<RadarEvent *> *_Nullable events, RadarUser *_Nullable user,
                                                          NSArray<RadarGeofence *> *_Nullable nearbyGeofences, RadarConfig *_Nullable config, NSString *_Nullable token) {
                                          self.sending = NO;
+                                         if (status != RadarStatusSuccess) {
```

Comment:
```suggestion
                                         if (status != RadarStatusSuccess || !config) {
```
Is there a reason why we are not checking here?

Reply:
Ah, indeed can check there.